devices/src/legacy/uart_pl011.rs

```diff
@@ -34,6 +34,7 @@ const UARTRIS: u64 = 15;
 const UARTMIS: u64 = 16;
```

Comment:
Nit: can you please update the first commit subject to `devices: Emulate debug I/O based on pl011`?

Reply:
Ok.